src/components/DropDown/index.tsx

```diff
@@ -0,0 +1,71 @@
+import cn from 'classnames';
+import React, { ChangeEvent } from 'react';
+import { Person } from '../../types/Person';
+
+type Props = {
+  onSelected: (person: Person) => void,
+  query: string,
+  isFocused: boolean,
+  setIsFocused: (isFocused: boolean) => void,
+  people: Person[] | undefined,
```

Comment:
You don't need to render the `DropDown` component if you don't have any people.
Also, if in typescript you expect that value can be undefined, you can do it like this `people?: Person[]`